packages/auth/src/handlers.ts

```diff
@@ -131,40 +131,7 @@ export const requests = {
       schemas: ['urn:ietf:params:scim:api:messages:2.0:PatchOp'],
       Operations: [
         {
-          op: 'add',
-          path: 'members',
-          value: [
-            {
-              display: user.userName, // The user's username
-              value: user.id, // The user's SCIM ID
-            },
-          ],
-        },
-      ],
-    }
-
-    const response = await api.patch(`/scim2/Groups/${role}`, roleUpdateData)
-
-    return updateGroupResponseSchema.safeParse(response.data)
-  },
-  removeWSO2UserRole: async (email: string, role: string) => {
-    const userResponse = await requests.getUser(email)
-
-    if (!userResponse.success) {
-      throw new Error(`Failed to retrieve user with email: ${email}`)
-    }
-
-    const user = userResponse.data.Resources?.[0]
-
-    if (!user) {
-      throw new Error(`No user found with email: ${email}`)
-    }
-
-    const roleUpdateData = {
-      schemas: ['urn:ietf:params:scim:api:messages:2.0:PatchOp'],
-      Operations: [
-        {
-          op: 'remove',
+          op: operation,
```

Comment:
Ensure this converts to the string value correctly is passing in the enum.
Should be fine.